test/doc/fuzzy/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+##==================================================================================================
```

Comment:
why is it empty ?

Reply:
because in the new module which contains only decorated old functions there is still no doc cpp